recipes/configs/dev/qwen3B_full_grpo.yaml

```diff
@@ -121,6 +121,16 @@ vllm:
   steps_before_sync: 1
   queue_maxsize: ${eval:'${vllm.num_workers} * ${steps_before_sync}'}
 
+reward_functions:
```

Comment:
Love it, looks super clear. I was wondering whether we would need a registry for these, but I'm convinced that `cfg.instantiate` and a path are good enough.

Reply:
The problem with registries is that it doesn't see custom user code unless it's on the import path, so it forces users to work in the code base instead of just build on top of the code base.